cpp/tests/io/parquet_test.cpp

```diff
@@ -4465,6 +4471,132 @@ TEST_F(ParquetWriterTest, CheckColumnOffsetIndexStruct)
   }
 }
 
+TEST_F(ParquetWriterTest, CheckColumnIndexListWithNulls)
+{
+  auto valids  = cudf::detail::make_counting_transform_iterator(0, [](auto i) { return i % 2; });
```

Comment:
Maybe a descriptive name for the masks, to make it easier to read the test cases below.
I think my suggested name is not great, feel free to pick a different name.
```suggestion
  auto null_at_odd  = cudf::detail::make_counting_transform_iterator(0, [](auto i) { return i % 2; });
```

Reply:
But...I just copied another test :cry: :rofl:  More seriously, yes, valid1/valid2 makes it hard to tell what's expected.  Will fix.